integration/kubernetes/helm/alluxio/templates/alluxio-master.yaml

```diff
@@ -70,9 +72,16 @@ spec:
         app: {{ $masterName }}
     spec:
       securityContext:
-        runAsUser: {{ $root.Values.user }}
-        runAsGroup: {{ $root.Values.group }}
         fsGroup: {{ $root.Values.fsGroup }}
+      {{- if and $isSingleUfsLocal (not $isRoot) }}
+      initContainers:
+      - name: journal-chown
+        image: busybox
```

Comment:
we should not create a dependency on a new image. modify the Dockerfile if needed to install `sudo` by `apk --no-cache --update add sudo`

Reply:
Turned out I cannot `sudo` in initContainer (maybe not even possible in container commands in general). So I had to specify root in initContainer as following.
```
securityContext:
  runAsUser: 0
```